assignments/pull_requests.go

```diff
@@ -0,0 +1,98 @@
+package assignments
+
+import (
+	"context"
+	"errors"
+	"strconv"
+	"strings"
+
+	pb "github.com/autograde/quickfeed/ag"
+	"github.com/autograde/quickfeed/database"
+	"github.com/autograde/quickfeed/scm"
+	"github.com/google/go-github/v35/github"
+	"go.uber.org/zap"
+)
+
+var ErrInvalidBody = errors.New("invalid pull request body")
```

Comment:
Suggest remove this and instead return separate, specific errors to make it easier to distinguish different problems. Such exported sentinel errors should be avoided unless you need to compare an error against the sentinel error.

Reply:
Part of the reason I wanted this custom error was so that I could check for it in tests.
I changed it so that I only check whether I expect an error or not.